[25.931596756] 2019-05-25 17:50:51,958 - INFO - launcher: Starting sanity checks
[26.929855347] 2019-05-25 17:50:51,959 - INFO - launcher: Checking for Python 3.5+
[26.929855347] 2019-05-25 17:50:51,959 - INFO - launcher: Checking console encoding
[26.929855347] 2019-05-25 17:50:51,959 - INFO - launcher: Setting console encoding to UTF-8
[26.929855347] 2019-05-25 17:50:51,959 - INFO - launcher: Ensuring we're in the right environment
[28.900146484] 2019-05-25 17:50:51,961 - INFO - launcher: Adding local bins/ folder to path
[527.548789978] 2019-05-25 17:50:52,459 - INFO - launcher: Required checks passed.
[528.909683228] 2019-05-25 17:50:52,461 - INFO - launcher: Optional checks passed.
[529.546737671] 2019-05-25 17:50:52,461 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[0.9826951026916504] 2019-05-25 17:50:52,914 - WARNING - musicbot.config | In config.py::MainThread(17204), line 120 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[0.9826951026916504] 2019-05-25 17:50:52,914 - INFO - musicbot.config | In config.py::MainThread(17204), line 131 in run_checks: Using i18n: config/i18n/en.json
[0.9856870174407959] 2019-05-25 17:50:52,917 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 226 in _setup_logging: Set logging level to INFO
[0.9877521991729736] 2019-05-25 17:50:52,919 - DEBUG - musicbot.json | In json.py::MainThread(17204), line 8 in __init__: Init JSON obj with config/i18n/en.json
[1.0116653442382812] 2019-05-25 17:50:52,943 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 91 in __init__: Starting MusicBot release-120519
[1.0129187107086182] 2019-05-25 17:50:52,945 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 97 in __init__: Loaded autoplaylist with 2530 entries
[4.7197244167327881] 2019-05-25 17:50:56,651 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 949 in on_ready: Connection established, ready to go.
[4.7197244167327881] 2019-05-25 17:50:56,651 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 340 in _cache_app_info: Caching app info
[4.9402856826782227] 2019-05-25 17:50:56,872 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 771 in _scheck_ensure_env: Ensuring data folders exist
[4.9472773075103760] 2019-05-25 17:50:56,879 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 787 in _scheck_server_permissions: Checking server permissions
[4.9482278823852539] 2019-05-25 17:50:56,880 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 791 in _scheck_autoplaylist: Auditing autoplaylist
[4.9482278823852539] 2019-05-25 17:50:56,880 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 795 in _scheck_configs: Validating config
[4.9492764472961426] 2019-05-25 17:50:56,881 - DEBUG - musicbot.config | In config.py::MainThread(17204), line 228 in async_validate: Validating options...
[4.9492764472961426] 2019-05-25 17:50:56,881 - DEBUG - musicbot.config | In config.py::MainThread(17204), line 242 in async_validate: Acquired owner id via API
[4.9492764472961426] 2019-05-25 17:50:56,881 - DEBUG - musicbot.bot | In bot.py::MainThread(17204), line 798 in _scheck_configs: Validating permissions config
[4.9502587318420410] 2019-05-25 17:50:56,882 - DEBUG - musicbot.permissions | In permissions.py::MainThread(17204), line 94 in async_validate: Validating permissions...
[4.9502587318420410] 2019-05-25 17:50:56,882 - DEBUG - musicbot.permissions | In permissions.py::MainThread(17204), line 98 in async_validate: Fixing automatic owner group
[4.9502587318420410] 2019-05-25 17:50:56,882 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 966 in on_ready: Connected: 571278996583743488/MusicEY#3184
[4.9532606601715088] 2019-05-25 17:50:56,885 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 974 in on_ready: Owner:     372052771798515723/4Dnk GT#8924

[4.9542143344879150] 2019-05-25 17:50:56,886 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 977 in on_ready: Guild List:
[4.9562046527862549] 2019-05-25 17:50:56,888 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 981 in on_ready:  - 4Dnk Support
[4.9582040309906006] 2019-05-25 17:50:56,890 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 981 in on_ready:  - 4Dnk GT
[4.9601986408233643] 2019-05-25 17:50:56,892 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 1037 in on_ready: Not bound to any text channels
[4.9621880054473877] 2019-05-25 17:50:56,894 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 1063 in on_ready: Not autojoining any voice channels
[4.9652049541473389] 2019-05-25 17:50:56,897 - WARNING - musicbot.bot | In bot.py::MainThread(17204), line 1105 in on_ready: Your config file is missing some options. If you have recently updated, check the example_options.ini file to see if there are new options available to you. The options missing are: {'disablenowplayingautomatic', 'dmnowplaying'}
[32.9506998062133789] 2019-05-25 17:51:24,882 - INFO - musicbot.bot | In bot.py::MainThread(17204), line 2680 in on_message: 372052771798515723/4Dnk GT#8924: !help
